src/integrationTest/resources/values/ese-extension-values.yaml

```diff
@@ -0,0 +1,24 @@
+global:
+  rbac:
+    pspEnabled: false
+
+operator:
+  image: hivemq/hivemq-operator:snapshot
+  imagePullPolicy: Never
+  logLevel: DEBUG
+  admissionWebhooks:
+    enabled: false
+
+hivemq:
+  image: hivemq/hivemq4-test:k8s-snapshot
```

Comment:
what version of hivemq is tested here? Is it the latest? 
Can we add a test with the 4.9.latest image as well?

Reply:
We will need a new card, this only tests locally (so the image is from the master branch) so we can test on the mac and from development